### ISSUE TITLE
I get SQLError when trying to execute DB.schema.hasTable

### ISSUE THREAD

**Issue by subdan** [GetBreeze/breeze-db#5]
Hello.

I want to check if table exists in a database. I use hasTable method to do it but I get this error:

`SQLError: 'Error #3115: SQL Error.', details:'The schema object with name 'children' of type 'table' in database 'main' was not found.', operation:'schema', detailID:'1007'`

How to fix it?

```
setupDB(checkTable);

function setupDB(callback:Function):void
{
	DB.setup(function (error:Error):void
	{
		if (error == null)
		{
			callback();
		}
		else
		{
			trace(error)
		}
	});
}

function checkTable():void
{
	DB.schema.hasTable("children", function (error:Error, hasTable:Boolean):void
	{
		if (error == null)
		{
			trace(hasTable);
		}
		else
		{
			trace(error);
		}
	});
}
```

**Comment by subdan** [comment]
Thanks, Marcel.

**Comment by marpies** [comment]
Thanks for reporting. The error is simply stating the table has not been found, but indeed it is confusing and should not be part of the response in this case. We have fixed that in [v1.1.2](https://github.com/GetBreeze/breeze-db/releases/tag/v1.1.2).
